pwm_min: 205
pwm_max: 410
pwm_midpoint: 308
